app/models/recruitment_cycle.rb

```diff
@@ -1,10 +1,16 @@
 module RecruitmentCycle
   CYCLES = {
-    '2022' => '2021 to 2022 (starts 2022)',
-    '2021' => '2020 to 2021 (starts 2021)',
-    '2020' => '2019 to 2020 (starts 2020)',
+    '2022' => '2021 to 2022',
+    '2021' => '2020 to 2021',
+    '2020' => '2019 to 2020',
+    '2019' => '2018 to 2019',
```

Comment:
did we have any courses in 2019?

Reply:
not having this was causing some of the support system specs to fail. I had some issues logging into support yesterday but just sorted it out so let me have a better look